deploy/deployConfig.go

```diff
@@ -11,6 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build !nodeploy
```

Comment:
What's the purpose of this?

Reply:
New format for build constraints with 1.17
<https://pkg.go.dev/go/build#hdr-Build_Constraints>